src/seedu/addressbook/AddressBook.java

```diff
@@ -210,13 +210,17 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        loopProgram();
+    }
+
+	private static void loopProgram() {
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.